cmd/kubemark/hollow-node.go

```diff
@@ -181,52 +168,84 @@ func run(config *hollowNodeConfig) {
 	if len(config.TenantServers) == 0 {
 		klog.V(3).Infof("TenantServers is not set. Default to single tenant partition and clientConfig setting")
 		config.TenantServers = make([]string, 1)
-		config.TenantServers[0] = clientConfigs.GetConfig().Host
+		config.TenantServers[0] = config.KubeconfigPath
 	}
 	if config.ResourceServer == "" {
 		klog.V(3).Infof("Resource is not set. Default to clientConfig setting")
-		config.ResourceServer = clientConfigs.GetConfig().Host
+		config.ResourceServer = config.KubeconfigPath
 	}
 
 	// initialize the kubeclient manager
 	kubeclientmanager.NewKubeClientManager()
 
+	// TODO: set this from the hollow node env variable
+	enableInSecurePorts := false
+
 	numberTenantPartitions := len(config.TenantServers)
 	clients := make([]clientset.Interface, numberTenantPartitions)
 	for i := 0; i < len(clients); i++ {
-		tenantCfg := restclient.CopyConfigs(clientConfigs)
-		for _, cfg := range tenantCfg.GetAllConfigs() {
-			cfg.Host = config.TenantServers[i]
-		}
-		clientFromConfig, err := clientset.NewForConfig(tenantCfg)
-		if err != nil {
-			klog.Fatalf("Failed to create a ClientSet: %v. Exiting.", err)
+		if enableInSecurePorts == false {
```

Comment:
I thought we use kubeconfig here for secure connection. But it looks like insecure mode we also change to kubeconfig? Looks like this code will be obsolete soon.

Reply:
from now on, kubeconfig will  be the media to start the arktos components. for insecure connection, the kubeconfig will be http://serverip:port.